buildSrc/build.gradle

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This configuration specifies gradle plugins for all Google Ads Java library
+ * subprojects.
+ */
+
+plugins {
+    id 'groovy-gradle-plugin'
```

Comment:
For my own edification... what does this do? Where/how is it used?

Reply:
Gradle build config files can be written in Groovy or Kotlin. Including this plugin imports the core Gradle dependency management and project configuration functionality.

Reply:
The project would not be able to build. This plugin enables the core Gradle behaviors using the Groovy language. The directory acts as the top-level management for the entire project..